app/components/amount-input-screen/amount-input-screen-ui.tsx

```diff
@@ -0,0 +1,188 @@
+import * as React from "react"
+import { useI18nContext } from "@app/i18n/i18n-react"
+import { makeStyles, Text } from "@rneui/themed"
+import { View } from "react-native"
+import { GaloyCloseButton, GaloyIconButton } from "../atomic/galoy-icon-button"
+import { GaloyPrimaryButton } from "../atomic/galoy-primary-button"
+import { GaloyWarning } from "../atomic/galoy-warning"
+import { CurrencyKeyboard } from "../currency-keyboard"
+import { Key } from "./number-pad-reducer"
+
+export type AmountInputScreenUIProps = {
+  primaryCurrencySymbol?: string
+  primaryCurrencyFormattedAmount?: string
+  primaryCurrencyCode: string
+  secondaryCurrencySymbol?: string
+  secondaryCurrencyFormattedAmount?: string
+  secondaryCurrencyCode?: string
+  errorMessage?: string
+  setAmountDisabled?: boolean
+  onKeyPress: (key: Key) => void
+  onToggleCurrency?: () => void
+  onClearAmount: () => void
+  onSetAmountPress?: () => void
+  goBack: () => void
+}
+
+export const AmountInputScreenUI = ({
+  primaryCurrencySymbol,
+  primaryCurrencyFormattedAmount,
+  primaryCurrencyCode,
+  secondaryCurrencySymbol,
+  secondaryCurrencyFormattedAmount,
+  secondaryCurrencyCode,
+  errorMessage,
+  onKeyPress,
+  onToggleCurrency,
+  onClearAmount,
+  onSetAmountPress,
+  setAmountDisabled,
+  goBack,
+}: AmountInputScreenUIProps) => {
```

Comment:
```suggestion
export const AmountInputScreenUI: React.FC<AmountInputScreenUIProps> = ({
  primaryCurrencySymbol,
  primaryCurrencyFormattedAmount,
  primaryCurrencyCode,
  secondaryCurrencySymbol,
  secondaryCurrencyFormattedAmount,
  secondaryCurrencyCode,
  errorMessage,
  onKeyPress,
  onToggleCurrency,
  onClearAmount,
  onSetAmountPress,
  setAmountDisabled,
  goBack,
}) => {
```

Reply:
this also ensure the return type to be compatible. I don't think it's critical but this is the pattern I've been using with the refactor I've done.

Reply:
I can make those changes.